public/app/plugins/datasource/mssql/specs/datasource.test.ts

```diff
@@ -335,3 +334,17 @@ describe('MSSQLDatasource', () => {
     });
   });
 });
+
+function createFetchResponse<T>(data: T): FetchResponse<T> {
```

Comment:
I see we use this function in Postgres datasource too, maybe you can extract it so a helper util?

Reply:
Yepp, I think I did that in another PR already. I was thinking about doing that and changing any observableTester usages to the new jest matcher at the same PR.